packages/truffle/test/scenarios/commands/develop.js

```diff
@@ -78,19 +78,19 @@ describe("truffle develop", function () {
     [
       {
         cmd: "console",
-        expectedError: `ℹ️ : 'console' is not allowed within Truffle REPL`
+        expectedError: `ℹ️ : 'truffle console' is not valid in Console environment.`
       },
       {
         cmd: "CONSOLE",
-        expectedError: `ℹ️ : 'CONSOLE' is not allowed within Truffle REPL`
+        expectedError: `ℹ️ : 'truffle CONSOLE' is not valid in Console environment.`
       },
       {
         cmd: "develop",
-        expectedError: `ℹ️ : 'develop' is not allowed within Truffle REPL`
+        expectedError: `ℹ️ : 'truffle develop' is not valid in Console environment.`
       },
       {
         cmd: "alakazam",
-        expectedError: `ℹ️ : 'alakazam' is not a valid Truffle command`
+        expectedError: `ℹ️ : 'truffle alakazam' is not valid in Console environment`
```

Comment:
Oh, here's that error message though...?  If this error message ever actually does come up, I feel like it's misleading to say it's "not valid in a console environment" when in fact it's not valid at all!

Reply:
Yea, that is misleading. :thinking:

Reply:
Addressed in prior comment.